ui/desktop/app/templates/application.hbs

```diff
@@ -21,25 +21,28 @@
 
       <header.utilities as |utilities|>
         {{#if this.showWindowActions}}
-          <Rose::Button
-            @iconOnly='flight-icons/svg/x-16'
-            @style='warning'
+          <Hds::Button
+            @icon='x'
+            @size='small'
+            @text={{t 'actions.close'}}
+            @color='critical'
```

Comment:
Note: Hds has 4 colors, and since `warning` is not available on btns I changed to `critical` but open to suggestions. See [docs](https://helios.hashicorp.design/components/button?tab=code#color)

Reply:
Or maybe `@color='tertiary'` is less alarming?

Reply:
Perhaps a question for design team